pom.xml

```diff
@@ -154,9 +175,9 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>org.mnode.ical4j</groupId>
-			<artifactId>ical4j</artifactId>
-			<version>1.0.4</version>
+			<groupId>org.bedework</groupId>
+			<artifactId>bw-ical4j-cl</artifactId>
+			<version>1.1.2-SNAPSHOT</version>
```

Comment:
Since we're working on releasing, don't depend on -SNAPSHOT versions. Change this to the published 1.1.1.